scarpe-components/lib/scarpe/components/calzini.rb

```diff
@@ -184,10 +184,21 @@ def spacing_styles_for_attr(attr, props, styles, with_options: true)
     end
   end
 
+  def first_color_of(*colors)
```

Comment:
I totally like this. I could see where maybe we'd want to start shoving some of this in color, but first_color_of is exactly what I was envisaging. What's the first non-nil color in the collection, make it a hex, spit it out. great!